lib/literal/array.rb

```diff
@@ -64,6 +64,12 @@ def &(other)
 		end
 	end
 
+  def ==(other)
+    other.is_a?(Literal::Array) && @__value__ == other.__value__
```

Comment:
Can we use `Literal::Array === other` here instead?

Reply:
Can do, but it breaks some other tests internally. Messages aren't clear so I'll dig into it. 🫡

Reply:
Sorry, Quickdraw is in a bit of a state at the moment. I put up with it and use it to encourage me to make Quickdraw better. 😅

Reply:
I just ran this and it works for me.

```ruby
def ==(other)
	Literal::Array === other && @__value__ == other.__value__
end
```

Reply:
My bad, I was doing something entirely different. Thanks!